jenkins/internal-ci/centos-7.9.2009/pr/cortx_deployment/motr.groovy

```diff
@@ -33,7 +36,7 @@ pipeline {
         MOTR_PR_REFSEPEC = "${ghprbPullId != null ? MOTR_GPR_REFSEPEC : MOTR_BRANCH_REFSEPEC}"
 
         //////////////////////////////// BUILD VARS //////////////////////////////////////////////////
-        // OS_VERSION, host and COMPONENTS_BRANCH are manually created parameters in jenkins job.
+        // OS_VERSION, singlenode_host, multinode_hosts and COMPONENTS_BRANCH are manually created parameters in jenkins job.
```

Comment:
Should s/multinode_hosts/3 node for now

Reply:
Added threenode_hosts this parameter name. As 3node_hosts this was getting some issue.